tracecat/db/schemas.py

```diff
@@ -505,6 +505,10 @@ class RegistryAction(Resource, table=True):
     display_group: str | None = Field(
         None, description="The presentation group of the action", nullable=True
     )
+    doc_url: str | None = Field(
+        None, description="Link to documentation", nullable=True
+    )
+    author: str | None = Field(None, description="Author of the action", nullable=True)
```

Comment:
if you explicitly use the default keyword like such `Field(default=None, ...)`, the type errors saying missing defaults will be resolved. Would be great if you can make these updates for the whole `RegistryAction` model as well